didcomm_messaging/legacy/messaging.py

```diff
@@ -0,0 +1,280 @@
+"""Legacy messaging service."""
+
+from dataclasses import dataclass
+import json
+from typing import Generic, Optional, Sequence, Union
+
+import base58
+from pydantic import AnyUrl
+from pydid import VerificationMethod
+from pydid.service import DIDCommV1Service
+
+from didcomm_messaging.crypto import P, S, SecretsManager
+from didcomm_messaging.legacy.base import LegacyCryptoService
+from didcomm_messaging.legacy.packaging import LegacyPackagingService
+from didcomm_messaging.resolver import DIDResolver
+
+
+class LegacyDIDCommMessagingError(Exception):
+    """Raised on error in legacy didcomm messaging."""
+
+
+@dataclass
+class LegacyPackResult:
```

Comment:
There's a small part of me that wonders if "Legacy" is the right term to use in reference to DIDComm V1. Obviously we hope and intend that DIDComm V2 will overtake DIDComm V1 soon, but it seems very much the case that DIDComm V1 is the most common production use case right now. 
I don't know that I have particularly strong feelings on the matter. I'm fine using "Legacy" as a...statement of intent? Using language that coincides with how we want things to be. I'm just not sure if it's strictly accurate to say that DIDComm V1 is currently legacy, if that makes sense